python/nemo_relay/scope_local.py

```
@@ -511,7 +550,9 @@ def deregister_llm_conditional_execution(scope_handle, name):
 # ---------------------------------------------------------------------------
 
 
-def register_llm_request(scope_handle, name, priority, break_chain, fn):
+def register_llm_request(
+    scope_handle: ScopeHandle, name: str, priority: int, break_chain: bool, fn: Callable[..., Any]
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)